src/components/HexInput.tsx

```diff
@@ -8,6 +8,7 @@ const escape = (hex: string) => hex.replace(/([^0-9A-F]+)/gi, "");
 interface Props {
   color: string;
   onChange: (newColor: string) => void;
+  [key: string]: unknown;
```

Comment:
Sorry, could've sworn I fixed that as I remember doing this.

```suggestion
interface Props extends HTMLInputElement {
  color: string;
  onChange: (newColor: string) => void;
```

Reply:
That looks much better, thanks!

Reply:
Apologies, I know I tried this and used it, just don't know what happened to it.